build/cpp/warble/inc/warble_functions.h

```diff
@@ -122,26 +117,31 @@ ReadReplyPackager(const reply_vector &result);
 //   int1: 1 indicating get
 //   string1: username, username_following
 //   string2: empty string
-const request_vector
-ProfileRequestPackager(const Any &any);
+const request_vector ProfileRequestPackager(const Any &any);
 
 // receives a vector containing the two vectors of the user's followers and
 // followings
 // @param: reply vector
 //   holds two vectors containing the username_followers and username_followings
 // @return: ProfileReply any
-const Any
-ProfileReplyPackager(const reply_vector &result);
+const Any ProfileReplyPackager(const reply_vector &result);
+
+// returns a list of clients that should be streamed given the streaming payload
+const std::vector<std::string> StreamHandler(const std::vector<std::pair<std::string, Any>> &clients, const std::string &warble);
 
 // the unordered map for associating function names with functions
 //  function names map to request function and reply function
-const function_mapping
-    func_map{
-        {"Registeruser", std::make_pair(RegisteruserRequestPackager,
-                                        RegisteruserReplyPackager)},
-        {"Warble", std::make_pair(WarbleRequestPackager, WarbleReplyPackager)},
-        {"Follow", std::make_pair(FollowRequestPackager, FollowReplyPackager)},
-        {"Read", std::make_pair(ReadRequestPackager, ReadReplyPackager)},
-        {"Profile",
-         std::make_pair(ProfileRequestPackager, ProfileReplyPackager)}};
-} // namespace warble_functions
\ No newline at end of file
+const function_mapping func_map{
+  {"Registeruser",
+    std::make_pair(RegisteruserRequestPackager, RegisteruserReplyPackager)},
+  {"Warble", std::make_pair(WarbleRequestPackager, WarbleReplyPackager)},
+  {"Follow", std::make_pair(FollowRequestPackager, FollowReplyPackager)},
+  {"Read", std::make_pair(ReadRequestPackager, ReadReplyPackager)},
+  {"Profile", std::make_pair(ProfileRequestPackager, ProfileReplyPackager)}
```

Comment:
You may have intended this, but Stream is not in this func_map

Reply:
Yes, there is a separate map used for stream events, defined in `warble_functions.h`